### PR TITLE
:tada: initial version of constants for kodadot

### PR DESCRIPTION
**Pull request by vikiival**
**Thank you for your contribution** to the [KodaDot NFT gallery](https://kodadot.xyz).
👇 \_ Let's make a quick check before the contribution.

### PR type

- [ ] Fix
- [x] Feature
- [x] Refactoring
- [ ] Chore

### What's new?

- [x] PR refs #102

### CODE REVIEW

**Comment by vikiival** [comment]
Please if I forgot something, pls let me know.
Or if you advise any other data type or so.

**Comment by roiLeo** [comment]
gh workflow won't work inside `static` should be at root level

Not sure if it's a good idea to centralized config in a fixed package version.

What would be the steps if we need to make a quick update on relay chain because one of them is dead?

**Comment by vikiival** [comment]
> gh workflow won't work inside static should be at root level

fixed in #111

> What would be the steps if we need to make a quick update on relay chain because one of them is dead?

Not sure what you mean?

**Comment by roiLeo** [comment]
> Not sure what you mean?

Let's say new squid rmrk2 (rc1) out, to make progress on developments we will have to wait for a new package release.
And what if now the squid endpoint broke then new package release to fallback previous endpoint, update the package version on marketplace and then update it again a few days later.

File: static/src/endpoints.ts [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+export const ENDPOINT_MAP: Config<WS_URL> = {
+  bsx: 'wss://basilisk-rpc.dwellir.com',
+  glmr: 'wss://public-rpc.pinknode.io/moonbeam',
+  ksm: 'wss://kusama-rpc.dwellir.com',
+  movr: 'wss://wss.api.moonriver.moonbeam.network',
+  rmrk2: 'wss://kusama-rpc.dwellir.com',
+  snek: 'wss://rococo-basilisk-rpc.hydration.dev',
+}
+
+// export const ALTERNATIVE_ENDPOINT_MAP: Config<WS_URL[]> = {}
```
Comment (Jarsen136) [inline static/src/endpoints.ts:14]:
```suggestion
```

File: static/test/index.test.ts [hunk 2]
```diff
@@ -0,0 +1,8 @@
+import { expect, it, describe } from "vitest";
+import {} from "../src";
+
+describe("packageName", () => {
+  it.todo("pass", () => {
+    expect(true).toBe(true);
```
Comment (Jarsen136) [inline static/test/index.test.ts:6]: Could we remove this test case? 😄

File: static/src/chains.ts [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+  2,
+  12,
+  'KSM',
+  'https://kusama.subscan.io/'
+)
+
+export const CHAINS: Config<ChainProperties> = {
+  ksm: DEFAULT_CHAIN_PROPERTIES,
+  rmrk2: DEFAULT_CHAIN_PROPERTIES,
+  bsx: toChainProperty(10_041, 12, 'BSX', 'https://basilisk.subscan.io/'),
```
Comment (Jarsen136) [inline static/src/chains.ts:27]:
```suggestion
  bsx: toChainProperty(10041, 12, 'BSX', 'https://basilisk.subscan.io/'),
```

**Comment by vikiival** [comment]
> Let's say new squid rmrk2 (rc1) out, to make progress on developments we will have to wait for a new package release.
And what if now the squid endpoint broke then new package release to fallback previous endpoint, update the package version on marketplace and then update it again a few days later.

The intention of the package is to make some constants `static` something that would not change. For development purposes I would definitely extend package on frontend.

Second things is that we have a ton of obsolete code on the frontend.

File: static/src/chains.ts (hunk 3 above)
Comment (vikiival) [reply static/src/chains.ts:27]: https://github.com/sindresorhus/eslint-plugin-unicorn/blob/v43.0.2/docs/rules/numeric-separators-style.md

**Review by Jarsen136 — APPROVED** [review]
LGTM

**Comment by vikiival** [comment]
@preschian @roiLeo if it makes more sense for you i can move it under the nft-gallery so it will be awailable directly in the workspace. 

I just need these constants across the other packages (uniquery, vuex-options)

WDYT?

**Pull request merged**